sdk/core/switcher/entry.S

```diff
@@ -607,6 +646,13 @@ exception_entry_asm:
 
 	// Load the trusted stack pointer to ct1
 	cspecialr          ct1, mtdc
+#ifdef CONFIG_MSHWM
+	// Update the spilled copy of the stack high watermark to ensure that we
```

Comment:
```suggestion
	// Update the spilled copy of the stack high water mark to ensure that we
```
Would be nice to be consistent to make searching easier.

Reply:
Technically, it should be high-water mark, but this is probably fine.